docker-compose.yml

```diff
@@ -0,0 +1,41 @@
+version: '2'
+
+services:
+  nginx:
+    image: bitnami/nginx:1.12.1-r1
+    volumes:
+      - ./dev/nginx:/bitnami/nginx/conf/vhosts
+    depends_on:
+      - backend
+      - frontend
+    environment:
+      - VIRTUAL_HOST=fortress.docker
+      - VIRTUAL_PORT=8080
+    ports:
+      - 8080:8080
+  mongo:
+    image: bitnami/mongodb:3.4.9-r1
+  backend:
+    image: angelrb/fortress:latest
```

Comment:
We may want to use `build: .` for dev and `image: ...` for production. So we can move easily to a CI system to build and push images to a docker registry.

This is a note, though. We don't need to think about it right now.

Reply:
I agree with you but it will depend on how we'll deploy the project. If we're going to deploy it using containers (Kubernetes or other platform), it's fine to move this image to `build`